resume.html

```diff
@@ -0,0 +1,171 @@
+<!DOCTYPE html>
+<html lang='en'>
+	<head>
+		<meta charset="utf-8">
+		<meta http-equiv="x-ua-compatible" content="ie=edge">
+		<meta name="viewport" content="width=device-width, initial-scale=1">
+
+		<title>wommy.io - resume</title>
+
+		<link rel='stylesheet' href='style.css'>
+		<style>
+			#main {
+				margin: 1rem;
+			}
+			h2 {
+				border-bottom: 1px solid silver;
+				padding-bottom: .3rem;
+			}
+			.margin{
+				margin: 1rem 0;
+			}
+			.indent{
+				padding-left: 1.25rem;
+			}
+			.highlight{
+				background-color: gainsboro;
+				border-radius: 3px;
+				padding: .2em .4em;
+				font-size: 85%;
+				font-family: monospace;
+			}
+		</style>
+	</head>
+	<body id='resume'>
+		<nav>
+			<a href='./index.html'>home</a>
+		</nav>
```

Comment:
How come this page has navigation but the other one doesn't?

Reply:
the index? because its entire purpose is to index the other pages; every element is navigation or a label to navigation

Reply:
the mcchris page doesn't have navigation

Reply:
on the index, every element serves as navigation or labeling navigation

Reply:
because mcchris is supposed to replace mcchris.com, im not going to put a link back to my page on his page